### PR TITLE
=rem #17440 Harden RemoteReDeploymentSpec

### PR DESCRIPTION
**Pull request by benmccann**
Backport of https://github.com/akka/akka/commit/bd280e325280e0e9c93fc499c6ce0cd76c24d10c#diff-6911a9effffad86984433d8d3ee7ff1d to fix test flakiness encountered in https://github.com/akka/akka/pull/18116

### CODE REVIEW

**Comment by patriknw** [comment]
OK TO TEST

**Comment by patriknw** [comment]
LGTM

File: akka-remote-tests/src/multi-jvm/scala/akka/remote/RemoteReDeploymentSpec.scala [hunk 1]
```diff
@@ -1,3 +1,6 @@
+/**
+ * Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+ */
```
Comment (ktoso) [inline akka-remote-tests/src/multi-jvm/scala/akka/remote/RemoteReDeploymentSpec.scala]: oh :)

**Comment by ktoso** [comment]
LGTM

**Pull request merged**